airflow/config_templates/config.yml

```diff
@@ -249,6 +249,14 @@
       type: string
       example: ~
       default: "0"
+    - name: default_task_retry_delay
+      description: |
+          The number of seconds each task is going to wait by default between retries. Can be overridden at
+          dag or task level.
+      version_added: 2.3.2
+      type: integer
+      example: ~
+      default: 300
```

Comment:
Without this setting the current behavior is 0 seconds for delay isn't it?
So what is the motivation to change it to 300?

Reply:
@eladkal in [abstractoperator](https://github.com/apache/airflow/blob/main/airflow/models/abstractoperator.py#L65) it gets set to 300 as of now, so if we keep it to 0 in default_config, the fallback won't going to work.

Reply:
Ah OK. Didn't know we have default of 300 :)